onnxruntime/test/util/include/test_utils.h

```diff
@@ -25,6 +25,8 @@ struct EPVerificationParams {
   // the default of 1e-5f, especially for scenarios such as [Q -> Quantized op -> DQ]
   // Set this only if this is necessary
   float fp32_abs_err = 1e-5f;
+
+  bool verify_expected_failure_graph{false};
```

Comment:
>verify_expected_failure_graph [](http://example.com/codeflow?start=7&length=29)

instead of both `verify_entire_graph_use_ep` and `verify_expected_failure_graph`, consider something like
`enum class ExpectedEPNodeAssignment { None, Some, All };`